src/views/ModelPlan/ReadOnly/Table/index.tsx

```diff
@@ -116,11 +115,27 @@ const Table = ({
       {
         Header: t('allModels.tableHeading.category'),
         accessor: 'basics.modelCategory',
-        Cell: ({ value }: any) => {
+        Cell: ({ row, value }: any) => {
+          const additionalModelCategory =
+            row.original.basics.additionalModelCategories;
+
           if (!value) {
             return <div>{h('noAnswer.tBD')}</div>;
           }
-          return translateModelCategory(value);
+          if (value) {
```

Comment:
This value check isn't necessary as it's already being checked in the line above